### PR TITLE
Added Github action to deploy plugin to Wordpress SVN

### PR DESCRIPTION
**Pull request by Lucasarenaterry**
For this to work we have to add the secrete keys: SVN_USERNAME & SVN_PASSWORD. We will be given once the plugin has been approved.

### CODE REVIEW

File: .github/workflows/deployToWordPressSVN.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Deploy to WordPress SVN
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  tag:
+    name: New tag
```
Comment (MattGibney) [inline .github/workflows/deployToWordPressSVN.yml:8]: Give this a name

File: .github/workflows/deployToWordPressSVN.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Deploy to WordPress SVN
+on:
+  push:
+    tags:
+    - "*"
```
Comment (MattGibney) [inline .github/workflows/deployToWordPressSVN.yml:5]: Update this to specifically run ONLY on Stable Releases (Not drafts and not tags).

File: .github/workflows/deployToWordPressSVN.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    - name: Build
+      run: |
+        npm install
+        npm run build
+    - name: WordPress Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@master
+      env:
+        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+        SLUG: code-syntax-block
```
Comment (MattGibney) [inline .github/workflows/deployToWordPressSVN.yml:21]: That's not our slug

File: .github/workflows/deployToWordPressSVN.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  tag:
+    name: New tag
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Build
+      run: |
+        npm install
+        npm run build
```
Comment (MattGibney) [inline .github/workflows/deployToWordPressSVN.yml:15]: This repo doesn't have any NodeJS. There are no deps to build.

**Review by MattGibney — CHANGES_REQUESTED** [review]

**Review by orssich — APPROVED** [review]

**Review by MattGibney — APPROVED** [review]

**Pull request merged**